src/main/java/cubicchunks/world/type/CustomCubicChunksWorldType.java

```diff
@@ -41,19 +46,35 @@ public static void create() {
 		new CustomCubicChunksWorldType();
 	}
 
-	@Override public ICubicChunkGenerator createCubeGenerator(ICubicWorldServer world) {
-		CubeProcessor terrain = new CustomTerrainProcessor(world);
-		CubeProcessor features = new CustomFeatureProcessor();
-		CubeProcessor population = new CustomPopulationProcessor(world);
-		return new ICubicChunkGenerator() {
-			@Override public void generateTerrain(Cube cube) {
-				terrain.calculate(cube);
-				features.calculate(cube);
-				cube.initSkyLight();
+	@Override
+	public WorldProvider getReplacedProviderFor(WorldProvider provider) {
+		return provider; // TODO: Custom Nether? Custom End????
+	}
+
+	@Override public ICubeGenerator createCubeGenerator(ICubicWorldServer world) {
+		CustomTerrainProcessor    terrain    = new CustomTerrainProcessor(world);
+		CustomFeatureProcessor    features   = new CustomFeatureProcessor();
```

Comment:
I guess it can be done. We can discuss universal eclipse/idea formatting settings later